en/ui-system/components/editor/ui-transform.md

```diff
@@ -36,3 +35,15 @@ export class Example extends Component {
         uiTransform.anchorY = 0.5;
     }
 }
+```
+
+### Deprecation of the priority property
+
+We deprecated the priority property of the UITransform component in version 3.1, and users can adjust the rendering order by setting the order of the node tree using setSiblingIndex.
+
+A note on the removal of the priority property and the recommended SiblingIndex property.
+The priority property on the UITransform component has been removed due to unclear ideation and naming conflicts with other properties in the engine, which was originally designed to provide a shortcut for the user to sort the node tree and has no other use in itself, and is not related to the priority expressed by priority.
```

Comment:
> 1. The priority property on the UITransform component has been removed due to unclear ideation and naming conflicts with other properties in the engine, which was originally designed to provide a shortcut for the user to sort the node tree and has no other use in itself, and is not related to the priority expressed by priority.

Reply:
We don't consider __ideation__ to be a positive word. However, I'm not sure what you intend to mean when writing it. Tell me and I can suggest a replacement.

Reply:
that mean the name of **priority** does not match and clear，which word can I use?

Reply:
can you use priority?
```suggestion
> 1. The priority property on the UITransform component has been removed due to unclear priority and naming conflicts with other properties in the engine, which was originally designed to provide a shortcut for the user to sort the node tree and has no other use in itself, and is not related to the priority expressed by priority.
```

Reply:
that's OK,thanks for your help.